installing/disconnected_install/installing-mirroring-installation-images.adoc

```diff
@@ -36,6 +36,11 @@ If you have an entitlement to Red Hat Quay, see the documentation on deploying R
 
 * If you do not already have an existing solution for a container image registry, subscribers of {product-title} are provided a xref:../../installing/disconnected_install/installing-mirroring-creating-registry.adoc#installing-mirroring-creating-registry[mirror registry for Red Hat OpenShift]. The _mirror registry for Red Hat OpenShift_ is included with your subscription and is a small-scale container registry that can be used to mirror the required container images of {product-title} in disconnected installations.
 
+[NOTE]
```

Comment:
Should this one be removed now since you put the note in the about the mirror registry file? It's showing up twice in this assembly now

Reply:
Addressed